benchexec/tablegenerator/__init__.py

```diff
@@ -697,6 +695,13 @@ def read_logfile_lines(log_file):
 
                     value = get_value_from_logfile(logfileLines, column.pattern)
 
+            if column.title.lower() == 'score':
+                if value is None:
+                    value = str(score)
+                else:
+                    # If a score column exists in the xml, we use that column's value for the score, instead of
+                    # the computed score.
+                    score = float(value)
```

Comment:
Nit-picking: I would actually prefer the branches the other way around:
`if value is not None:` and then
as comment "If a score value does not exist in the XML, then we use the computed score."

Reply:
I prefer the branching that way because "value is None" is a simpler expression than "value is not None".
But I adjusted the comment as you suggested, since it better describes the real logic behind it.